DQMOffline/Trigger/python/SusyMonitor_cfi.py

```diff
@@ -0,0 +1,105 @@
+hltSUSYmonitoring.histoPSet.jetEtaBinning = cms.vdouble(-2.4,-2.1,-1.5,-0.9,-0.3,0.,0.3,0.9,1.5,2.1,2.4)
+hltSUSYmonitoring.histoPSet.muEtaBinning  = cms.vdouble(-2.4,-2.1,-1.5,-0.9,-0.3,0.,0.3,0.9,1.5,2.1,2.4)
+#pt binning
+hltSUSYmonitoring.histoPSet.elePtBinning = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
+hltSUSYmonitoring.histoPSet.jetPtBinning = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
+ltTOPmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
+#Eta binning 2D
+hltSUSYmonitoring.histoPSet.eleEtaBinning2D = cms.vdouble(-2.5,-1.5,-0.6,0.,0.6,1.5,2.5)
+hltSUSYmonitoring.histoPSet.jetEtaBinning2D = cms.vdouble(-2.5,-1.5,-0.6,0.,0.6,1.5,2.5)
+hltSUSYmonitoring.histoPSet.muEtaBinning2D  = cms.vdouble(-2.5,-1.5,-0.6,0.,0.6,1.5,2.5)
+#pt binning 2D
+hltSUSYmonitoring.histoPSet.elePtBinning2D = cms.vdouble(0,20,30,50,100,200,400)
+hltSUSYmonitoring.histoPSet.jetPtBinning2D = cms.vdouble(0,20,30,50,100,200,400)
+hltSUSYmonitoring.histoPSet.muPtBinning2D  = cms.vdouble(0,20,30,50,100,200,400)
+#HT and phi binning 2D
+hltSUSYmonitoring.histoPSet.HTBinning2D  = cms.vdouble(0,20,40,70,100,150,200,400,700)
+hltSUSYmonitoring.histoPSet.phiBinning2D = cms.vdouble(-3.1416,-1.8849,-0.6283,0.6283,1.8849,3.1416)
+
+
+hltSUSYmonitoring.met       = cms.InputTag("pfMetEI") # pfMet
+hltSUSYmonitoring.jets      = cms.InputTag("ak4PFJetsCHS") # ak4PFJets, ak4PFJetsCHS, pfJetsEI
+hltSUSYmonitoring.electrons = cms.InputTag("gedGsfElectrons") # while pfIsolatedElectronsEI are reco::PFCandidate !
+hltSUSYmonitoring.muons     = cms.InputTag("muons") # while pfIsolatedMuonsEI are reco::PFCandidate !
+#Suvankar
+hltSUSYmonitoring.vertices  = cms.InputTag("offlinePrimaryVertices")
+
+# Marina
+hltSUSYmonitoring.btagalgo  = cms.InputTag("pfCombinedSecondaryVertexV2BJetTags")
+hltSUSYmonitoring.workingpoint     = cms.double(0.8484) # Medium
+
+hltSUSYmonitoring.HTdefinition = cms.string('pt>30 & abs(eta)<2.5')
+hltSUSYmonitoring.leptJetDeltaRmin = cms.double(0.4)
+
+hltSUSYmonitoring.numGenericTriggerEventPSet.andOr         = cms.bool( False )
+hltSUSYmonitoring.numGenericTriggerEventPSet.andOrHlt      = cms.bool(True)# True:=OR; False:=AND
+hltSUSYmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults::HLT" )
+hltSUSYmonitoring.numGenericTriggerEventPSet.errorReplyHlt = cms.bool( False )
+hltSUSYmonitoring.numGenericTriggerEventPSet.verbosityLevel = cms.uint32(0)
+
+hltSUSYmonitoring.denGenericTriggerEventPSet.andOr         = cms.bool( False )
+hltSUSYmonitoring.numGenericTriggerEventPSet.andOrHlt      = cms.bool(True)# True:=OR; False:=AND
+hltSUSYmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults::HLT" )
+hltSUSYmonitoring.numGenericTriggerEventPSet.errorReplyHlt = cms.bool( False )
+hltSUSYmonitoring.denGenericTriggerEventPSet.dcsInputTag   = cms.InputTag( "scalersRawToDigi" )
```

Comment:
@parbol lines 97-99 should be "denGenericTriggerEventPSet". I know the bug comes from my code (which should be fixed now), sorry about this